lua/dap-disasm.lua

```diff
@@ -326,6 +326,12 @@ M.config = {
     "instructionBytes",
     "instruction",
   },
+  dapview = {
+    keymap = "D",
+    label = "Disassembly [D]",
+    -- nerd font icon nf-md-cog
```

Comment:
Taking the opportunity, could you append this comment to the next line?

Reply:
I meant to make it one line: `short_label = "󰒓 [D]", -- nerd font icon nf-md-cog`